src/components/geocoder/geocoder.js

```diff
@@ -49,6 +50,17 @@ export const testForCoordinates = query => {
   return [isValid, Number(tokens[0]), Number(tokens[1])];
 };
 
+export const fixMeridian = bbox => {
```

Comment:
please write a function that returns a new `bbox`, instead of directly editing it